content/en/docs/Upgrade to PHP 8.1/_index.md

```diff
@@ -0,0 +1,483 @@
+
+```
+$ sudo nano ~/.profile
+```
+
+Add the following lines to the bottom of the file.
+
+```
+export LD_LIBRARY_PATH="$LD_LIBRARY_PATH:/opt/IBM/Informix_Client-SDK/lib:/opt/IBM/Informix_Client-SDK/lib/esql" 
+export INFORMIXDIR="/opt/IBM/Informix_Client-SDK"
+```
+
+Make sure `pdo_informix.so` is built.
+
+```
+$ find /usr/lib/php/20210902/pdo_informix.so
+```
+
+If `pdo_informix.so` exists, connect it to PHP's modules.
+
+```
+$ echo 'extension=pdo_informix.so' | sudo tee /etc/php/8.1/mods-available/pdo_informix.ini
+$ sudo phpenmod -v 8.1 -s ALL pdo_informix
+```
+
+### LDAP & Active Directory
+**Likelihood Of Impact: Low**
+
+DreamFactory uses [Lightweight Directory Access Protocol](https://www.php.net/manual/en/book.ldap.php) to lookup information or devices within a network. Therefore, to continue working with these services, you need to install the LDAP [extension](https://www.php.net/manual/en/ldap.installation.php) for the new version of PHP. For a quick setup, see the ['Installing the PHP extension'](#installing-the-php-extension) section.
+
+### MQTT
+**Likelihood Of Impact: Low**
+
+Previously, the MQTT service relied on the [Mosquitto-PHP](https://github.com/mgdm/Mosquitto-PHP) extension, which in turn relies upon the [libmosquitto](https://mosquitto.org/) library. From now on, this service no longer depends on the system's needs and contains everything you need to work in itself.
+
+### Oracle
+**Likelihood Of Impact: High**
+
+{{< alert color="success" title="Tip" >}}
+This section shows the minimum number of steps to set up required dependencies so that DreamFactory can work with Oracle. For a complete setup, follow the [official guide](https://www.oracle.com/database/technologies/releasenote-odbc-ic.html).
+{{< /alert >}}
+
+To use the Oracle service, you need to install the new version of the `oci8` extension compatible with PHP 8.1. You can seamlessly install the required extension version you need via [pecl](https://pecl.php.net/package/oci8). But before installation, you need to go through a few more steps:
+
+Download the desired Instant Client and the corresponding SDK Package. We will show the example of RPM packages.
+
+https://www.oracle.com/database/technologies/instant-client/downloads.html
+
+Install the necessary dependencies.
+
+```
+$ sudo apt install -y alien libaio1
+```
+
+Install the Oracle client packages. This may take a few minutes.
+
+```
+$ sudo alien -i oracle-instantclient-basic-21.9.0.0.0-1.el8.x86_64.rpm
+$ sudo alien -i oracle-instantclient-devel-21.9.0.0.0-1.el8.x86_64.rpm
+```
+
+Now we are good to install the `oci8` extension. The latest version is compatible with PHP 8.1 at present, but you may need to specify the version explicitly.
+
+```
+$ sudo pecl install oci8 (php 8.1 exactly what we need)
+$ sudo pecl install oci8-2.2.0 (php 7)
+```
+
+Once complete, make sure that OCI is loaded:
+
+```
+$ php -m | grep oci8
+
+odi8
```

Comment:
if this is expected output, lets use a comment `#` for it to prevent an error after pasting snippet into terminal.